scripts/build-bootstraps.sh

```diff
@@ -119,7 +119,7 @@ extract_debs() {
 	for deb in *.deb; do
 
 		current_package_name="$(echo "$deb" | sed -E 's/^([^_]+).*/\1/' )"
-		current_package_arch="$(echo "$deb" | sed -E 's/.*_(.*).deb$/\1/' )"
+		current_package_arch="$(echo "$deb" | sed -E 's/.*_(aarch64|all|arm|i686|x86_64).deb$/\1/' )"
```

Comment:
Honestly I initially want to change the regex just like that :-)

However I thought there was possibly a reason not just listing all 4 supported architectures plus `all`? Maybe I just overthought on this.

And... I still think as long as the package name and version name are guaranteed to not have any underscore, the `x86_64` placed at the end of full file name is possibly not a problem. *(however I'm not going to put modified regex again here)*